app/controllers/api/drops_controller.rb

```diff
@@ -2,6 +2,12 @@ class Api::DropsController < ApplicationController
   respond_to :json
 
   def index
-    @drops = Drop.all
+    if params[:latitude] && params[:longitude]
+      radius = params[:radius].presence || 10
+      places_within_radius = Place.near([params[:latitude], params[:longitude]], radius, units:  :km)
+      @drops = Drop.where(place_id: places_within_radius.map(&:id))
```

Comment:
you can also use a super awesome method here called pluck - which it a but faster than `map`.
ex: `@drops = Drop.where(place_id: places_within_radius.pluck(:id))`